pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>org.springframework.data</groupId>
 	<artifactId>spring-data-envers</artifactId>
-	<version>0.3.0.BUILD-SNAPSHOT</version>

```

Comment:
No need to upgrade the version number manually as we do this after a release. 0.3.0.BUILD-SNAPSHOT is the correct version. Please remove from PR.